app/Http/Requests/UserUpdateRequest.php

```diff
@@ -29,16 +29,26 @@ public function rules(): array
         return [
             'name' => ['required', 'string', 'max:255', new NoBlankCharacters],
             'username' => [
-                'required', 'string', 'min:4', 'max:50', Rule::unique(User::class)->ignore($user->id),
+                'required',
+                'string',
+                'min:4',
+                'max:50',
+                Rule::unique(User::class)->ignore($user->id),
                 new Username($user),
             ],
             'email' => [
-                'required', 'string', 'lowercase', 'email', 'max:255', Rule::unique(User::class)->ignore($user->id),
+                'required',
+                'string',
+                'lowercase',
+                'email',
+                'max:255',
+                Rule::unique(User::class)->ignore($user->id),
                 new UnauthorizedEmailProviders(),
             ],
             'mail_preference_time' => [Rule::enum(UserMailPreference::class)],
             'bio' => ['nullable', 'string', 'max:255'],
             'prefers_anonymous_questions' => ['required', 'boolean'],
+            'prefers_questions' => ['nullable', 'boolean'],
```

Comment:
Your migration doesn't set the `prefers_questions` as nullable, so why should this be nullable?

Reply:
will update the line. Thanks for pointing out.